docs/preview/features/azurefunctions-http-template.md

```diff
@@ -31,11 +31,16 @@ Creates a starter worker project with by default configured:
     * Request content and header to restrict to JSON content and JSON parsing with data model annotations validation
     * General exception handling that results in 500 Internal Server Error
     * OpenAPI docs generation and UI (see more info [here](https://github.com/Azure/azure-functions-openapi-extension))
+    * Dockerfile
 
 ### Configuration
 
 And additional features available with options:
 
+* `--functions-worker`: Configures the type of Azure Functions worker type the project should target.
+  * `inProcess` (default): Uses the in-process Azure Functions worker type which runs on the same process as run Azure Function
```

Comment:
I wonder if we should stick with inProcess as the default option.
When looking at the [roadmap](https://techcommunity.microsoft.com/t5/apps-on-azure-blog/net-on-azure-functions-roadmap/ba-p/2197916), it looks as if .NET 6 will be the last version of .NET that supports in-process Functions.

Reply:
Oops, I think that in this PR, we use `isolated` as default in the Azure Functions HTTP trigger, but forgot to update the docs.
The other Azure Functions are still using `inProcess` as default, though.

Reply:
Updated this one.

Reply:
For the other ones: https://github.com/arcus-azure/arcus.templates/issues/748